src/common/components/vanilla/heading/index.js

```diff
@@ -4,15 +4,17 @@ import styles from './heading.css';
 
 const Heading = (props) => {
   const H = props.heading || 'h1';
+  const A = props.align || 'left';
   return (
-    <H className={ styles[H] }>
+    <H className={ `${styles[H]} ${styles[A]} `}>
       { props.children }
     </H>
   );
 };
 
 Heading.propTypes = {
   children: PropTypes.node,
+  align: React.PropTypes.oneOf(['left', 'right', 'center']),
```

Comment:
I agree... we don't really ever do anything other than left align, but jujucharms added center... we should look at this again.